src/frontend/src/hooks/change-requests.hooks.ts

```diff
@@ -13,9 +13,14 @@ import {
   getSingleChangeRequest,
   reviewChangeRequest,
   addProposedSolution,
-  deleteChangeRequest
+  deleteChangeRequest,
+  requestCRReview
 } from '../apis/change-requests.api';
 
+export interface CRReviewPayload {
+  userIds: number[];
```

Comment:
imo this type is so simple, that making it a type is less helpful than just writing it out wherever it's used, so I'd replace wherever this type is used with just `{
  userIds: number[];
}`